repo/packages/chips/package.py

```diff
@@ -0,0 +1,39 @@
+from spack.package import *
+
+class Chips(MakefilePackage):
+
+    homepage = "https://github.com/cathryntrott/chips2024"
+    # git = "https://github.com/cathryntrott/chips2024.git"
+    git = "https://github.com/d3v-null/chips2024.git"
+
+    maintainers('d3v-null')
+
+    version("main", branch="main")
```

Comment:
In addition to the "main" branch you could specify a particular commit, just to have the option of a reproducible installation (main branch can be updated).